wdc-calculator-core/wdc-calculator-core-action/src/main/java/hu/icellmobilsoft/wdc/calculator/core/action/WorkdayDataReader.java

```diff
@@ -100,19 +103,33 @@ private WorkdayData getWorkdayDataFromFile(File file) throws BusinessException {
 
     private void readIncludeData() throws BusinessException {
         try {
-            Arrays.stream(calculatorCoreConfig.getIncludeDays().split(DATE_DELIMITER)).filter(d -> !d.trim().isEmpty())
-                    .forEach(d -> workdayCache.put(LocalDate.parse(d), true));
-        } catch (DateTimeParseException e) {
+            readData(calculatorCoreConfig.getIncludeDays(), true);
+        } catch (DateTimeParseException | IllegalArgumentException e) {
             throw new BusinessException(ReasonCode.INVALID_INPUT, "Include data could not be read. " + e.getMessage(), e);
         }
     }
 
     private void readExcludeData() throws BusinessException {
         try {
-            Arrays.stream(calculatorCoreConfig.getExcludeDays().split(DATE_DELIMITER)).filter(d -> !d.trim().isEmpty())
-                    .forEach(d -> workdayCache.put(LocalDate.parse(d), false));
-        } catch (DateTimeParseException e) {
+            readData(calculatorCoreConfig.getExcludeDays(),false);
+        } catch (DateTimeParseException | IllegalArgumentException e) {
             throw new BusinessException(ReasonCode.INVALID_INPUT, "Exclude data could not be read. " + e.getMessage(), e);
         }
     }
+
+    private void readData(String data, boolean isWorkingDay) throws BusinessException {
+        for (String entry : data.split(DATE_DELIMITER)) {
+            if (!entry.trim().isEmpty()) {
+                String[] fields = entry.split(FIELD_DELIMITER);
+                if (fields.length == 1) {
+                    workdayCache.put(LocalDate.parse(fields[0]), isWorkingDay, null, null, null);
```

Comment:
ezeket a feldogozasokat egyertelmuve kell tenni ha elszalas tortenik. Pontosan melyik sorban mit nem lehet feldolgozni, parszolni. Aki ala teszi a fajlt annak kell tudnia hol van a hiba

Reply:
Clarified message, it now contains the invalid entry.